packages/fiori/cypress/specs/DynamicSideContent.cy.tsx

```diff
@@ -80,3 +81,398 @@ describe("Accessibility", () => {
 			.should("have.attr", "aria-label", customSideContentLabel);
 	});
 });
+
+describe("'sideContentPosition' property", () => {
+	it("set to 'End'", () => {
+		cy.mount(
+			<DynamicSideContent sideContentPosition="End">
+				<div>
+					<h1>Main Content</h1>
+				</div>
+				<div slot="sideContent">
+					<h1>Side Content</h1>
+				</div>
+			</DynamicSideContent>
+		);
+		cy.get("[ui5-dynamic-side-content]").as("dynamicSideContent");
+
+		cy.get("@dynamicSideContent").shadow().find(".ui5-dsc-root > *").first()
+			.should("have.class", "ui5-dsc-main");
+	});
+
+	it("set to 'Start'", () => {
+		cy.mount(
+			<DynamicSideContent sideContentPosition="Start">
+				<div>
+					<h1>Main Content</h1>
+				</div>
+				<div slot="sideContent">
+					<h1>Side Content</h1>
+				</div>
+			</DynamicSideContent>
+		);
+		cy.get("[ui5-dynamic-side-content]").as("dynamicSideContent");
+
+		cy.get("@dynamicSideContent").shadow().find(".ui5-dsc-root > *").first()
+			.should("have.class", "ui5-dsc-side");
+	});
+});
+
+describe("'equalSplit' property", () => {
+	it("set to 'true'", () => {
+		cy.mount(
+			<DynamicSideContent equalSplit>
+				<div>
+					<h1>Main Content</h1>
+				</div>
+				<div slot="sideContent">
+					<h1>Side Content</h1>
+				</div>
+			</DynamicSideContent>
+		);
+		cy.get("[ui5-dynamic-side-content]").as("dynamicSideContent");
+
+		cy.get("@dynamicSideContent").shadow().find(".ui5-dsc-main").should("have.class", "ui5-dsc-span-6");
```

Comment:
Not sure if its a convention, but I see across our tests that we chain the commands in a column-like way, e.g:

```ts
cy.get("@dynamicSideContent")
	.shadow()
	.find(".ui5-dsc-main")
	.should("have.class", "ui5-dsc-span-6");
```

Can we do this everywhere in the this test as well for consistency ?